mlflow_faculty/trackingstore.py

```diff
@@ -199,7 +201,21 @@ def create_run(
 
         :return: The created Run object
         """
-        raise NotImplementedError()
+        try:
+            faculty_run = self._client.create_run(
+                self._project_id,
+                experiment_id,
+                datetime.fromtimestamp(start_time / 1000, tz=UTC),
+            )
+        except faculty.clients.base.HttpError as e:
+            raise MlflowException(
+                "{}. Received response {} with status code {}".format(
```

Comment:
Should we have something more specific e.g. 'Failed to create run.' as part of the error message?

Reply:
How about creating a custom exception class, e.g. `FacultyStoreException` that takes 2 constructor arguments
 - message (e.g. 'Failed to create run')
 - a `faculty.clients.base.HttpError`

and extends `MlflowException`

and does the `"{}. Received response {} with status code {}".format()...` inside ?

There are other store-specific exceptions in the mlflow codebase, such as `RestException`.

Reply:
I'm not sure if this is super valuable beyond just having the error message inside a standard MlflowException. I would advocate using MlflowException for now and subclassing later if it is considered valuable.

Reply:
We could have helper method instead, that just instantiates an MlflowException, but I gues one of my points was to do the `"{}. Received response {} with status code {}"` on one common place and the using more descriptive error messages on the specific places.

Reply:
I'm going to construct the exception manually for now. If we re-use similar code in different places, we can refactor it later.